versions/1/integration/elasticsearch/transform/manifest.spec.yml

```diff
@@ -0,0 +1,14 @@
+##
+## Describes the specification for a transform's manifest.yml file
+##
+spec:
+  type: object
+  additionalProperties: false
+  definitions:
+    index_template:
+      $ref: "../../data_stream/manifest.spec.yml#/properties/elasticsearch/properties/index_template"
+  properties:
```

Comment:
Didn't we have a requirement to have the option to disable starting a transform by default? I'd be ok doing it in an immediate follow up PR to just get this one in first.

Reply:
Updated the spec to include `start`, added an example, updated the description on the spec & issue.